Docs/source/developers/qed.rst

```diff
@@ -0,0 +1,39 @@
+.. _developers-qed:
+
+QED
+====================
+
+Quantum synchrotron
+-------------------
+
+.. note::
+
+   Section empty!
+
+Breit-Wheeler
+-------------
+
+.. note::
+
+   Section empty!
+
+Schwinger process
+-----------------
+
+If the code is compiled with QED and the user activates the Schwinger process in the input file,
+electron-positron pairs can be created in vacuum in the function
+``MultiParticleContainer::doQEDSchwinger`` which is called at every time step in ``Evolve``.
+``MultiParticleContainer::doQEDSchwinger`` in turn calls the function ``filterCreateTransformFromFAB``,
+which proceeds in three steps.
```

Comment:
Could you try how this looks? And if you like it, feel free to add similar changes for other functions you mention.
```suggestion
If the code is compiled with QED and the user activates the Schwinger process in the input file,
electron-positron pairs can be created in vacuum in the function

.. doxygenclass:: MultiParticleContainer::doQEDSchwinger

which is called at every time step in ``Evolve``. ``MultiParticleContainer::doQEDSchwinger`` in turn calls the function

.. doxygenclass:: filterCreateTransformFromFAB

which proceeds in three steps.
```

Reply:
Actually the doxygen doc does not include functions defined under `#ifdef WARPX_QED` so this doesn't work. Should we change that first? I can write a mini-PR if it's just adding `WARPX_QED = 1` to the `PREDEFINED` tag in the `Doxyfile`.

Reply:
Yes I think it is worth doing this, you can include it in this PR.

Reply:
Ok I've done this. That's a cool feature indeed! The doxygen doc for `filterCreateTransformFromFAB` is a bit lengthy with all the arguments and templates arguments but I guess it's ok since it's the developers doc.